src/ODEmodel.jl

```diff
@@ -69,7 +69,7 @@ function predict(p, g_0::Real, t, nG1::Integer, nG2::Integer, nD1, nD2)
     end
 
     v = [ones(nG1) * p[5] * g_0 / nG1; ones(nG2) * (1.0 - p[5]) * g_0 / nG2; D1; D2]
```

Comment:
If you want to run the model with one drug then another, what you'll need to change is the initial vector after the switch. This is an initial value problem—you can't solve the ODE except by starting at t=0.

Reply:
Sorry, yes, I missed it. I Will work on it.

Reply:
I thought it could start from any given time if we have the information at that exact time. In here, I am calling predict again, and giving it the last time point of the previous drug as initial values `g0`:
![image](https://user-images.githubusercontent.com/34812750/77273418-c5451f80-6c70-11ea-9854-90673a7cb6cc.png)

Isn't it taking care of this part you are mentioning?

Reply:
But the next predict can't just use the g1 and g2 quantities. It needs the entire v vector.

Reply:
Sorry, I thought the `v` is being updated inside the predict here:
![image](https://user-images.githubusercontent.com/34812750/77277148-8a93b500-6c79-11ea-9a87-fb17e5c2cd2a.png)

You mean I should input the v vector to the `predict`?

Reply:
Yeah, you'll need to output this and be able to provide it to a predict function. It's the actual starting point for the ODE system. G1 and G2 are summary quantities.

Reply:
Why don't you write a new, separate version of predict() that just takes a single time point and gives you back the full species vector, using Krylov solving (`ExponentialUtilities.jl`)? It's imperative that you understand what this function is doing.

Reply:
Thank youuu, I will do that.